src/main/kotlin/com/vk/kphpstorm/KphpStormParserDefinition.kt

```diff
@@ -35,6 +35,9 @@ class KphpStormParserDefinition() : PhpParserDefinition() {
             KphpDocTplParameterDeclPsiImpl.elementType    -> KphpDocTplParameterDeclPsiImpl(node)
             KphpDocElementTypes.kphpDocTagWarnPerformance -> KphpDocTagWarnPerformancePsiImpl(node)
             KphpDocWarnPerformanceItemPsiImpl.elementType -> KphpDocWarnPerformanceItemPsiImpl(node)
+            KphpDocElementTypes.kphpDocTagJson            -> KphpDocTagJsonPsiImpl(node)
+            KphpDocJsonPropertyPsiImpl.elementType        -> KphpDocJsonPropertyPsiImpl(node)
```

Comment:
Here and everywhere: not "property", but "attrubite" (or just "attr") — to match with PHP/KPHP

Reply:
done